website/static/js/fangorn.js

```diff
@@ -1280,6 +1291,13 @@ function _fangornResolveRows(item) {
     });
 
     if (item.data.kind === 'file') {
+      default_columns.push(
+      {
+        data: 'version',
```

Comment:
What are you using for an editor or IDE? It should be able to use the project's .jshintrc to know how many spaces there are per tab in a javascript file so you don't have to deal with it manually. See if you can get that fixed and then go ahead and clean up the spaces from this commit.